component/vshn_postgres.jsonnet

```diff
@@ -829,6 +829,7 @@ local composition(restore=false) =
               data: {
                 imageTag: common.GetAppCatImageTag(),
                 sgNamespace: pgParams.sgNamespace,
+                emailAlertingEnabled: std.toString(params.services.vshn.emailAlerting.enabled),
```

Comment:
is this param used anywhere?

Reply:
Yes in the linked PR: https://github.com/vshn/appcat/pull/27/files#diff-0831ba77cb39a636ab6a3bf8e7d398a672f4d916c9af0d78c36e5a60031d273aR138-R141